src/sentry/api/serializers/models/release.py

```diff
@@ -325,11 +325,15 @@ def __get_release_data_no_environment(self, project, item_list):
 
         first_seen: dict[str, datetime.datetime] = {}
         last_seen: dict[str, datetime.datetime] = {}
-        tag_values = tagstore.backend.get_release_tags(
-            organization_id,
-            project_ids,
-            environment_id=None,
-            versions=[o.version for o in item_list],
+        tag_values = (
+            tagstore.backend.get_release_tags(
+                organization_id,
+                project_ids,
+                environment_id=None,
+                versions=[o.version for o in item_list],
+            )
+            if no_snuba is False
+            else []
```

Comment:
nit: IMO standard if is easier to read here:


```suggestion
        if no_snuba: 
            tag_values = []
        else:
             tag_values = tagstore.backend.get_release_tags(
                organization_id,
                project_ids,
                environment_id=None,
                versions=[o.version for o in item_list],
            )
```